Tools/Scripts/webkitpy/xcode/simulated_device.py

```diff
@@ -448,6 +449,9 @@ def initialize_devices(cls, requests, host=None, name_base='Managed', simulator_
         deadline = time.time() + timeout
         for device in SimulatedDeviceManager.INITIALIZED_DEVICES:
             cls._wait_until_device_is_usable(device, deadline)
+            if apple_additions() and getattr(apple_additions(), 'disable_extra_simulator_logging', None) is not None:
+                _log.debug('Disabling extra logging for {}'.format(device))
+                apple_additions().disable_extra_simulator_logging(device)
```

Comment:
Callee will probably need the `port` or `host` object to interact with the executive or filesystem objects. Probably means the `host` object is the one you want.

Reply:
`device` contains an `executive` property.